lib/module-resolvers/file-resolver.js

```diff
@@ -69,7 +105,14 @@ class FileResolver {
     _resolve(module_path, module_name, importer_path) {
         var self = this;
 
-        var found = self._search(module_path, module_name, importer_path);
+        var found = false;
+        if (module_path.startsWith('./') ||
+            module_path.startsWith('../') ||
+            path.isAbsolute(module_path)) {
```

Comment:
Maybe put the condition into a helper method like `_is_local_import` for more clarity?